internal/hmyapi/apiv2/util.go

```diff
@@ -16,20 +16,21 @@ const (
 	defaultPageSize = uint32(100)
 )
 
-// ReturnWithPagination returns result with pagination (offset, page in TxHistoryArgs).
-func ReturnWithPagination(hashes []common.Hash, args TxHistoryArgs) []common.Hash {
+// ReturnPagination returns pagination params.
+func ReturnPagination(length int, args TxHistoryArgs) (int, TxHistoryArgs) {
```

Comment:
please add unit test for the pagination function.

Reply:
Ok